cms/static/js/personalise.js

```diff
@@ -61,10 +62,51 @@ app.ports.listeners.subscribe(function(res) {
     proConListeners();
     seeMoreListener();
     feedbackLoopListener();
+    swipeListeners();
   });
 });
 
 app.ports.selectTag.subscribe(function(tags) {
   localStorage.setItem('selected_tags', JSON.stringify(tags));
   app.ports.updateTags.send(tags);
 });
+
+function swipe(el, callback){
```

Comment:
could we use http://package.elm-lang.org/packages/kennib/elm-swipe/latest/Swipe ? The vanilla version looks good :+1: